src/main.ts

```diff
@@ -16,13 +16,20 @@ const contextPromise = store.dispatch('fetchContext').catch(() => {
   window.location.href = '/login'
 })
 
+const app:Vue = new Vue({ store, router, render: h => h(App) })
```

Comment:
why do we need this to 'add email and username' , suggest to add this to the commit msg, or even beter create a separate PR

Reply:
I moved it to a separate commit.